src/webapp/js/components/health-workers-form.js

```diff
@@ -174,24 +160,11 @@ class HealthWorkersForm extends Component {
   constructor(props) {
     super(props);
 
-    this.state = {
-      availableLocations: [],
-    };
-
     this.renderField = this.renderField.bind(this);
   }
 
   componentWillMount() {
-    this.fetchLocations();
-  }
-
-  fetchLocations() {
-    const url = '/api/districts';
-
-    apiClient.get(url).then((response) => {
-      const availableLocations = response.data;
-      this.setState({ availableLocations });
-    });
+    this.props.fetchLocations();
```

Comment:
Should we fetch locations on every load of Health Worker form? Maybe we can find some outer component, to make some initial actions there. If locations don't change a lot. What do you think?

Reply:
I was thinking about this, we can talk more tomorrow on standup